packages/amplify-migration-tests/src/configure/index.ts

```diff
@@ -0,0 +1,51 @@
+import * as nexpect from 'nexpect';
```

Comment:
use amplify-e2e-core instead of directly nexpect.

Reply:
using e2e-core